vendor/assets/stylesheets/components/text.scss

```diff
@@ -0,0 +1,11 @@
+span.error {
+  color: $c-bittersweet;
+}
+
+.uk-text-primary {
+  color: $c-outer-space !important;
+}
+
+.uk-text-brand-xl {
+  font-size: 7rem;
```

Comment:
This is not a UI Kit class. Just make `cmsify-branding` classes so they don't conflict and don't need `important`